typescript/cli/src/commands/core.ts

```diff
@@ -34,6 +43,48 @@ export const coreCommand: CommandModule = {
   handler: () => log('Command required'),
 };
 
+/**
+ * Generates a command module for deploying Hyperlane contracts, given a command
+ *
+ * @param commandName - the deploy command key used to look up the deployFunction
+ * @returns A command module used to deploy Hyperlane contracts.
+ */
+export const deploy: CommandModuleWithWriteContext<{
+  chain: string;
+  config: string;
+  dryRun: string;
+  fromAddress: string;
+}> = {
+  command: 'deploy',
+  describe: 'Deploy Hyperlane contracts',
+  builder: {
+    chain: chainCommandOption,
+    config: outputFileCommandOption(
```

Comment:
should this be input ?

Reply:
yes

Reply:
Will address in a following PR. Requires refactoring of the input function, but also merging this for bug bash